e2e/tests/interchain_accounts/localhost_test.go

```diff
@@ -185,6 +185,305 @@ func (s *LocalhostInterchainAccountsTestSuite) TestInterchainAccounts_Localhost(
 		s.AssertValidTxResponse(txResp)
 	})
 
+	t.Run("verify tokens transferred", func(t *testing.T) {
+		balance, err := chainA.GetBalance(ctx, userBWallet.FormattedAddress(), chainADenom)
+		s.Require().NoError(err)
+
+		expected := testvalues.IBCTransferAmount + testvalues.StartingTokenAmount
+		s.Require().Equal(expected, balance)
+	})
+}
+
+func (s *LocalhostInterchainAccountsTestSuite) TestInterchainAccounts_ReopenChannel_Localhost() {
+	t := s.T()
+	ctx := context.TODO()
+
+	// relayer and channel output is discarded, only a single chain is required
+	_, _ = s.SetupChainsRelayerAndChannel(ctx)
+	chainA, _ := s.GetChains()
```

Comment:
Maybe we should create an issue to add a second setup function which only spins up a single chain, could be useful for localhost/non-ibc e2e tests.

Reply:
Yeah, definitely agree!